File: ExtremeCore/Classes and Modules/serializeableColor.vb
Imports System.Drawing

Public Class SerializeableColor
    Public Property R As Integer
    Public Property G As Integer
    Public Property B As Integer
    Public Property A As Integer

    Public Function GetColor() As Color
        Return Color.FromArgb(A, R, G, B)
    End Function

    Public Sub SetColor(clr As Color)
        A = clr.A
        R = clr.R
        G = clr.G
        B = clr.B
    End Sub
End Class

File: ExtremeStudio.Core/Classes and Modules/serializeableColor.cs
using System.Windows.Media;

namespace ExtremeStudio.Core.Modules
{
	public class SerializeableColor
	{
		public byte R {get; set;}
		public byte G {get; set;}
		public byte B {get; set;}
		public byte A {get; set;}
		
		public Color GetColor()
		{
			return Color.FromArgb(A, R, G, B);
		}
		
		public void SetColor(Color clr)
		{
			A = clr.A;
			R = clr.R;
			G = clr.G;
			B = clr.B;
		}
	}
	
}
